rollup.config.js

```diff
@@ -0,0 +1,34 @@
+import node from "rollup-plugin-node-resolve";
+import {terser} from "rollup-plugin-terser";
+import * as meta from "./package.json";
+
+const copyright = `// ${meta.homepage} Version ${meta.version} Copyright ${(new Date).getFullYear()} ${meta.author.name}.`;
+
+export default [
```

Comment:
Maybe I'm misreading this, but it looks like both d3-axis.js and d3-axis.min.js will be compressed with this configuration?

Reply:
Oops, fixed!